Assignment2/Extract_Bits/ConsoleApp1/Program.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+
+/// <summary>
+/// Gets the number, start and stop bits then extracts the bits and provides the decimal value of it.
+/// </summary>
+class BitsExtraction{
+
+    /// <summary>
+    /// This Main method gets the inputs from the user.
+    /// </summary>
+    static void Main(){
+        Console.WriteLine("Enter the input number:");
+        var userInput = int.Parse(Console.ReadLine());
```

Comment:
consider changing parse to tryparse. it can handle corner cases.

Reply:
Modified to TryParse.